The only remaining error is:
```
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.37.32822\include\immintrin.h(15): fatal error C1189: #error:  This header is specific to X86, X64, ARM64, and ARM64EC targets
```

This occurs in ARM64 debug mode, release works fine. I am looking for a solution.

--- a/ports/anari/vcpkg.json
+++ b/ports/anari/vcpkg.json
@@ -4,6 +4,7 @@
   "description": "Cross-Platform 3D Rendering Engine API.",
   "homepage": "https://www.khronos.org/anari",
   "license": "Apache-2.0",
+  "supports": "!(arm64&windows)",
   "dependencies": [
     "glm",
     "python3",

--- a/ports/anari/vcpkg.json
+++ b/ports/anari/vcpkg.json
@@ -4,7 +4,7 @@
   "description": "Cross-Platform 3D Rendering Engine API.",
   "homepage": "https://www.khronos.org/anari",
   "license": "Apache-2.0",
-  "supports": "!(arm64&windows)",
+  "supports": "!(arm64 & windows)",
   "dependencies": [
     "glm",
     "python3",

--- a/versions/a-/anari.json
+++ b/versions/a-/anari.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "cf756ca650e9d72324024441ad48b760fb587f86",
+      "git-tree": "b4d46d64feb15c77963c539eac613274295d644b",
       "version": "0.7.0",
       "port-version": 0
     }

